templates/20-validation-webhook.deployment.yaml.tmpl

```diff
@@ -8,6 +8,9 @@ metadata:
   namespace: "#NAMESPACE#"
 spec:
   replicas: 3
+  selector:
+    matchLabels:
+      app: validation-webhook
```

Comment:
Just a minor thing, there is a placeholder for that, that gets replaced with the service name: `"#SVCNAME#"`

Reply:
Updated.